RUFAS/routines/field/field/field.py

```diff
@@ -1097,7 +1097,8 @@ def _record_planting(
         units = {
             "crop": MeasurementUnits.UNITLESS,
             "heat_scheduled_harvest": MeasurementUnits.UNITLESS,
-            "date": {"year": MeasurementUnits.CALENDAR_YEAR, "day": MeasurementUnits.ORDINAL_DAY},
+            "planting_year": MeasurementUnits.CALENDAR_YEAR,
+            "planting_day": MeasurementUnits.ORDINAL_DAY,
```

Comment:
```suggestion
            "year": MeasurementUnits.CALENDAR_YEAR,
            "day": MeasurementUnits.ORDINAL_DAY,
```
I think it would be better to just call these the year and day, because they're in `_record_planting` I think it is clear that they are the planting year and date.